examples/jenkins/jenkins-ephemeral-template.json

```diff
@@ -99,6 +101,14 @@
                   {
                     "name": "JENKINS_PASSWORD",
                     "value": "${JENKINS_PASSWORD}"
+                  },
+                  {
+                    "name": "KUBERNETES_MASTER",
```

Comment:
i'm not sure if it's still needed or not, at one point the kubernetes plugin needed this env variable, and "KUBERNETES_MASTER" is not one of the env variables that gets set in a pod automatically as far as i know.